src/gbt_convertors.pyx

```diff
@@ -152,7 +152,8 @@ def get_gbt_model_from_xgboost(booster: Any) -> Any:
     else:
         is_regression = True
 
-    n_iterations = int(len(trees_arr) / (n_classes if n_classes > 2 else 1))
+    n_iterations = booster.best_ntree_limit

```

Comment:
I think we should use [booster.best_iteration](https://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.XGBRegressor.best_iteration), not 
booster.best_ntree_limit.
From [this part of the code](https://github.com/dmlc/xgboost/blob/663136aa08c00598d8b49adf5901e4cb2ce187da/python-package/xgboost/core.py#L1143) I concluded, that best_ntree_limit will be removed soon.

Please take into account that best_iteration is starts from 0, while best_ntree_limit starts from 1. Thus we will have a bias equals to 1.